src/features/tasks/taskSlice.ts

```diff
@@ -0,0 +1,37 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+interface Task {
+  text: string;
+  completed: boolean;
+}
+
+interface RootState {
+  tasks: Task[];
+}
```

Comment:
Get used to move interfaces and types outside files with logic and import them wherever you need